tests/tests_output/powfacpy_single_file.py

```diff
@@ -0,0 +1,2539 @@
+"""Contains the class PFBaseInterface (interaction with the PF database)
```

Comment:
I don't understand the context of this everything-in-one file here

Reply:
This is an output of the test test_merge_powfacpy_package_into_single_file in test_script_interface.py.
However, these output files of tests are not required to keep track of with git, so I will not commit them. Actually I only wanted to commit the new tests_output folder.